packages/projects/src/components/Page.jsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) Kernel
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { Link } from 'react-router-dom'
+
+import { Footer, Navbar } from '@kernel/common'
+
+import AppConfig from 'App.config'
+
+const Page = ({ projectHandle, children }) => {
+  let additionalMenuItems
+  if (projectHandle) {
```

Comment:
It's often difficult to read long `if/else` statements so we try to simplify the cognitive overhead. We can simplify this by

1) Pulling out the Link component into its own function
```
const editMenuItem = ({ projectHandle }) => <Link ...>Edit</Link>
```

2) use a ternary expression
```
const additionalMenuItems = projectHandle ? [editMenuItem({ projectHandle })] : []
```